### PR TITLE
Onboarding presentation placeholder

### LINKED ISSUE #15: Prepare placeholder for new, onboarding, presentation

### PR DESCRIPTION
**Pull request by magul**
Fixes #15

### CODE REVIEW

**Comment by w1stler** [comment]
Shall I do some more steps to test this one to test locally? Links are failing on redirection in `index.html` to both sections.

File: onboarding.html [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+                <section>
+                    <section>
+                        <h1>
+                            <img src="img/code_for_poznan_square.png" style="background:none; border:none; box-shadow:none; "/>
+                        </h1>
+                    </section>
+                </section>
+                <section>
+                    <section>
+                        <h1>Soft introduction</h1>
```
Comment (w1stler) [inline onboarding.html:37]: Do we want any sections below all H1s?

Comment (magul) [reply onboarding.html:37]: That's just a placeholder, there are separate tickets to fill these places.

**Comment by magul** [comment]
> Shall I do some more steps to test this one to test locally? Links are failing on redirection in index.html to both sections.

That's not going to work without HTTP server running (so no access using `file://` protocol). To test it locally you can simply start a Python-based HTTP server in that directory (depending on Python version it's either `python -m SimpleHTTPServer` or `python3 -m http.server`) and go to `http://localhost:8000`. There these links should work properly.

**Pull request merged**